MaintSnaps.sh

```diff
@@ -22,8 +22,9 @@ TZ=zulu
 
 # Put the bulk of our variables into an external file so they
 # can be easily re-used across scripts
+# shellcheck source=/dev/null
 source "${PROGDIR}/commonVars.env"
-source "${PROGDIR}/setcred.sh" && PROGNAME="$( basename "${BASH_SOURCE[0]}" )"
+source PROGNAME="$( basename "${BASH_SOURCE[0]}" )"
```

Comment:
Remove `source` here, that was part of the setcred command

```suggestion
PROGNAME="$( basename "${BASH_SOURCE[0]}" )"
```